.gitignore

```diff
@@ -75,3 +75,5 @@ cscope.*
 *.swp
 *~
 
+# Visual Studio Code
+.vscode/
```

Comment:
This seems reasonable. We could get this in a lot faster if this was a separate PR.